### ISSUE TITLE
[BUG] - Pulse sheet viewer not showing content.

### ISSUE THREAD

**Issue by GlaserN** [zhinst/laboneq#28]
**Describe the bug**
The pulse sheet viewer for the following experiment shows nothing.

The debugging console shows the following error:

```
TypeError: Cannot read properties of undefined (reading 'id')
    at Ie (test_2023-03-03-14-34-59.html:54368:1160251)
    at Fe (test_2023-03-03-14-34-59.html:54368:1162037)
    at Object.useReducer (test_2023-03-03-14-34-59.html:54368:1048368)
    at t.useReducer (test_2023-03-03-14-34-59.html:54368:1123256)
    at Ht (test_2023-03-03-14-34-59.html:54368:1171271)
    at Ao (test_2023-03-03-14-34-59.html:54368:1041968)
    at Ts (test_2023-03-03-14-34-59.html:54368:1098927)
    at xc (test_2023-03-03-14-34-59.html:54368:1087909)
    at gc (test_2023-03-03-14-34-59.html:54368:1087837)
    at mc (test_2023-03-03-14-34-59.html:54368:1087700)
dl @ test_2023-03-03-14-34-59.html:54368
``` 

**To Reproduce**
Compile the following experiment and 
view the pulse sheet viewer:
[experiment_pulse_sheet.zip](https://github.com/zhinst/laboneq/files/10882442/experiment_pulse_sheet.zip)
[test_2023-03-03-14-38-41.html.zip](https://github.com/zhinst/laboneq/files/10882450/test_2023-03-03-14-38-41.zip)

**Versions used:**
 - laboneq: 2.1.0

**Comment by QuantumClemens** [comment]
@GlaserN thanks for reporting, will look into it

**Comment by polwel** [comment]
Hi Niklas, we have been aware of the issue, and have a fix ready. Unfortunately, it barely missed the deadline for the release last week. It will be part of release 2.2.

In the mean time, if you're interested, here's the updated pulse sheet viewer. Just [extract the HTML file](https://github.com/zhinst/laboneq/files/10899583/pulse_sheet_viewer_template.zip), and place it in your L1Q installation directory under `laboneq/pulse_sheet_viewer`.

(sending a zip because github won't let me attach HTML directly)

**Comment by QuantumClemens** [comment]
This bug is fixed in version 2.2 - closing this issue
